hypothesis-python/src/hypothesis/searchstrategy/strategies.py

```diff
@@ -268,6 +274,14 @@ def example(self, random=not_set):
         if random is not not_set:
             note_deprecation("The random argument does nothing", since="2019-07-08")
 
+        if getattr(sys, "ps1", None) is None:  # pragma: no branch
```

Comment:
Am I right in thinking this isn't covered because the process spawned by pexpect doesn't contribute information to coverage?

If so – it's worth a note here explaining that this branch is tested, it's just limitations of coverage reporting that mean we need this pragma.

If not – it definitely needs a comment to explain what I've misunderstood. :D

Reply:
You're right and I've added that comment :smile: